lightwood/helpers/accuracy.py

```diff
@@ -1,4 +1,38 @@
+from typing import Iterable, List
 from sklearn.metrics import r2_score as sk_r2_score
+from sklearn.metrics import f1_score as sk_f1_score
+from sklearn.metrics import recall_score as sk_recall_score
+from sklearn.metrics import precision_score as sk_precision_score
+
+
+def to_binary(y: Iterable) -> List[int]:
```

Comment:
is "y" always a list of True/False? If so, can you not just do a map like `{False: 0, True: 1}` or is it because lightwood converts categorical (including binary) info into a str type?

Reply:
The issue is that possible values for false are:
`'false'`, `'FALSE'`, `False`, `'False'`,`0` and `'0'`... and due to pd arbitrary type casting we can't guarantee what is what and due to user variability in how they declare false/true string we can't either (same for true labels)

This jus standardized that

We could do a map here, but I think the iteration is more obvious since it's a slightly complex operation, not just a simple map it's basically `lower -> map if -> cast else`